examples/views/tabular-input.php

```diff
@@ -37,7 +45,7 @@
         ],
         [
             'name' => 'title',
-            'title' => 'Title',
+            //'title' => 'Title',
```

Comment:
why did you update the tabular-input example file instead of create new file for list-input example?

Reply:
I checked the markup without a title. But it can be returned or comment added

Reply:
![image](https://user-images.githubusercontent.com/19654261/41296047-6d440ff8-6e75-11e8-9afb-0ab0bede3e4e.png)